src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5427,6 +5427,21 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_ImplicitCopyInReadOnlyMember_Title" xml:space="preserve">
     <value>Call to non-readonly member from a 'readonly' member results in an implicit copy.</value>
   </data>
+  <data name="ERR_StaticMemberCantBeReadOnly" xml:space="preserve">
+    <value>Static member '{0}' cannot be 'readonly'.</value>
```

Comment:
Let's give a little more info here, like: "readonly members cannot modify 'this' and static members do not have a 'this' reference"